android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt

```diff
@@ -100,6 +100,7 @@ constructor(
   val navigationConfiguration: NavigationConfiguration by lazy {
     configurationRegistry.retrieveConfiguration(ConfigType.Navigation)
   }
+  var syncStartedAt = OffsetDateTime.now()
```

Comment:
Can we have this in another PR if this functionality is needed.

Reply:
ok